tests/integration/utils/readers/test_base_reader.py

```
@@ -0,0 +1,149 @@
+
+        f = tmp_path / "toobig.txt"
+        f.write_bytes(b"x" * 1024)
+        with pytest.raises(FileTooLargeError, match="File too large to process"):
+            _check_file_size(f, max_bytes=512)
+
+    def test_error_message_contains_file_path(self, tmp_path: Path) -> None:
+        from file_organizer.utils.readers._base import FileTooLargeError, _check_file_size
+
+        f = tmp_path / "pathcheck.txt"
+        f.write_bytes(b"x" * 1024)
+        with pytest.raises(FileTooLargeError, match="pathcheck.txt"):
```

Ruff (0.15.9)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)